rewindable-index/test/Spec.hs

```diff
@@ -26,6 +27,20 @@ ixProperties = testGroup "Basic model"
       withMaxSuccess 1000 $ Ix.prop_insertRewindNotifications @Int @Int @Int Ix.conversion
   ]
 
+sProperties :: TestTree
+sProperties = testGroup "New index properties."
+  [ testProperty "New: Positive or non-positive depth" $
+      withMaxSuccess 10000 $ Ix.prop_observeNew @Int @Int S.conversion
```

Comment:
Should we increase the depth range for this to make sure we test closer to the actual depth value? The max depth for tests is 100, where is the indexers value is 2160

Reply:
I cannot see any scenario in which that would make a difference. I think that most issues will be found at boundaries. Making the boundaries be far apart or closer together would not influence that, would it?